packages/web/src/js/TextAreaGrow.ts

```diff
@@ -0,0 +1,69 @@
+import BaseComponent from './BaseComponent';
+import { enableToggleAutoloader } from './utils';
+import SelectorEngine from './dom/SelectorEngine';
+import EventHandler from './dom/EventHandler';
+
+const NAME = 'textAreaGrow';
+const DATA_KEY = 'textAreaGrow';
+const DATA_TEXTAREA = `data-${DATA_KEY}`;
+const RESIZE_EVENT = 'resize';
+const DEBOUNCE_DELAY = 50;
+
+function debounce<T extends (...args: never[]) => unknown>(func: T, wait: number): (...args: Parameters<T>) => void {
```

Comment:
Please, move this to somewhere else. E. g. utils directory or something similar.